Buildings/Experimental/OpenBuildingControl/CDL/Continuous/RampLimiter.mo

```diff
@@ -0,0 +1,93 @@
+within Buildings.Experimental.OpenBuildingControl.CDL.Continuous;
+block RampLimiter "Limit the increase or decrease rate of input"
+
+  parameter Real Increase "Increase amount of the input";
+  parameter Real Decrease "Decrease amount of the input";
+
+  parameter Modelica.SIunits.Time  IncDT(min=100*Constants.eps) = 1.0
+    "Amount of time between increases of input";
+  parameter Modelica.SIunits.Time  DecDT(min=100*Constants.eps) = 1.0
+    "Amount of time between decreases of input";
+
+  parameter Modelica.SIunits.Time Td(min=Constants.eps) = 0.001
+    "Derivative time constant";
+
+  parameter Boolean LimitsOn = true
+    "= false, if limits are off.";
+
+  Interfaces.RealInput u "Connector of Real input signal"
+    annotation (Placement(transformation(extent={{-140,-20},{-100,20}})));
+
+  Interfaces.RealOutput y "Connector of Real output signal"
+    annotation (Placement(transformation(extent={{100,-10},{120,10}})));
+
+protected
+  Real Rising = Increase/IncDT "Increase rate limit";
+  Real Falling = -Decrease/DecDT "Decrease rate limit";
+  Real val = (u-y)/Td;
+
+initial equation
+    y = u;
+
+equation
+  if LimitsOn then
+    der(y) = if val<Falling then Falling else if val>Rising then Rising else val;
+  else
+    y = u;
+  end if;
+   annotation (
+Documentation(info="<html>
+<p>
+The block limits the increase/decrease rate of its input signal in the range of <code>[Falling, Rising]</code>, where:
+</p>
+<pre>
+    <code>Falling = -Decrease/DecDT</code>; <code>Rising = Increase/IncDT</code>;
```

Comment:
Don't use <code> instead the <pre> section, as <pre> already typesets it as code.